Gemfile.lock

```diff
@@ -47,6 +47,7 @@ GEM
     parallel (1.17.0)
     parser (2.6.3.0)
       ast (~> 2.4.0)
+    private_address_check (0.4.1)
```

Comment:
Name: private_address_check
Version: 0.4.0
Advisory: CVE-2018-3759
Criticality: Unknown
URL: https://github.com/jtdowney/private_address_check/commit/4068228187db87fea7577f7020099399772bb147
Title: private_address_check Ruby Gem Time-of-check Time-of-use race condition
Solution: Upgrade to >= 0.5.0.